spec/factories/appliance_types.rb

```diff
@@ -23,6 +23,13 @@
       virtual_machine_templates { [ build(:virtual_machine_template)] }
     end
 
+    # If no os_family has been specified by the requestor, bind this appliance to the first available os_family
+    after(:build) do |atype|
+      if atype.os_family.blank?
+        atype.os_family = Atmosphere::OSFamily.first
```

Comment:
seed is a good idea; guarding against removal of last record is super paranoid - and the same logic applies to `Atmosphere::ComputeSite` and probably several other classes I can't think of right now. ;)